content/docs/capabilities/metrics.md

```diff
@@ -1,79 +1,266 @@
+
+![A chart displaying request duration in Pomerium Zero](./img/metrics/zero-request-duration-second-chart.png)
+
+</TabItem>
+<TabItem value="enterprise" label="Enterprise">
+
+Request duration measures the amount of time it takes Pomerium to proxy a request in milliseconds (ms). Pomerium Enterprise provides two request duration charts:
+
+The first chart organizes requests by duration ranges defined along the x-axis. Pomerium sums the total value of requests within each range and calculates the amount as a percentage value.
+
+![A chart displaying request duration in Pomerium Enterprise](./img/metrics/enterprise-request-duration.png)
+
+The second chart organizes requests by percentile ranges, date, and time. You can select a specific percentile range to view more granular data in that range:
+
+![A chart displaying request duration with percentile ranges in Enterprise Console](./img/metrics/enterprise-percentile-ranges.gif)
+
+</TabItem>
+</Tabs>
+
+### Requests rate and response codes
+
+<Tabs>
+<TabItem value="zero" label="Zero">
+
+The **Requests per second** chart calculates the average amount of proxied requests per second over the span of an hour. Requests are organized by date and time, and categorized by the following response status codes:
+
+- **200s** (200-299): successful responses
+- **300s** (300-399): redirection messages
+- **400s** (400-499): client error responses
+- **500s** (500-599): server error responses
+
+![A chart displaying requests per second in Pomerium Zero](./img/metrics/zero-requests-per-second.png)
+
+</TabItem>
+<TabItem value="enterprise" label="Enterprise">
+
+The **Request Rate** chart calculates the average amount of proxied requests per second over the span of an hour.
+
+![A chart displaying requests per second in Pomerium Enterprise](./img/metrics/enterprise-request-rate.png)
+
+The **Response Codes** chart organizes requests by date and time, and categorizes them with the following response status codes:
+
+- **HTTP 2xx** (200-299): successful responses
+- **HTTP 3xx** (300-399): redirection messages
+- **HTTP 4xx** (400-499): client error responses
+- **HTTP 5xx** (500-599): server error responses
+
+![A response codes chart in Pomerium Enterprise](./img/metrics/enterprise-response-status-codes.gif)
+
+</TabItem>
+</Tabs>
+
+### Bytes sent and received, request size
+
+<Tabs>
+<TabItem value="zero" label="Zero">
+
+The **Bytes sent** and **Bytes received** charts display the average amount of bytes sent and received over the span of an hour.
+
+![A chart displaying bytes sent and received in Pomerium Zero](./img/metrics/zero-bytes-sent-received.png)
+
+</TabItem>
+<TabItem value="enterprise" label="Enterprise">
+
+The **Request Size** chart organizes requests based on their request size measured in kilobytes (KB).
+
+![A chart displaying response size in Pomerium Enterprise](./img/metrics/enterprise-request-size.png)
+
+</TabItem>
+</Tabs>
+
+## Changesets and Deployments
```

Comment:
this section is definitely unrelated to Metrics, and probably has to be moved to its own subsection.

Reply:
right I think now that we merged _metrics_ content together for zero and enterprise its great. 

Maybe lets call the new section Changesets?